guake/paths.py.in

```diff
@@ -94,3 +94,5 @@ AUTOSTART_FOLDER = {{ AUTOSTART_FOLDER }}
 def try_to_compile_glib_schemas():
     log.info("Compiling schema: %s", SCHEMA_DIR)
     subprocess.check_call(["glib-compile-schemas", "--strict", SCHEMA_DIR])
+    with open(os.path.join(SCHEMA_DIR, "version"), "w", encoding="utf-8") as version:
+        version.write(guake.guake_version())
```

Comment:
Can this detect `dirty` version?

e.g. when you are doing some scheme change, will this embed something like `3.7.1-ea39dkf-dirty` before you bump Guake version?

Reply:
guake_version() in the dev branch appends ".dev[numbers]" to the end of the version number so there is distinction, but this is mostly moot for work in dev because `sudo make install` correctly creates the schema file always before we even get to this line, this patch only really affects people trying to install from pip or wheels, which only have clean version numbers.

Reply:
thanks for clarify